<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>1662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Identifiant&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;numero de telephone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>275</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Genre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Ville&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Etat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="identifiant">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numtel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="masculin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>masculin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>feminin</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ville">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Tunis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Beja</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gafsa</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="inscrit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inscription</string>
   </property>
  </widget>
  <widget class="QListWidget" name="chance_list">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>231</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="affiche_chance">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>afficher chance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="affiche_gagnant">
   <property name="geometry">
    <rect>
     <x>922</x>
     <y>30</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher gagnant</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableau">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>481</y>
     <width>631</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Identifiant</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>num tlf</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Genre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Etat</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ajouter">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="affiche_client">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher client</string>
   </property>
  </widget>
  <widget class="QListWidget" name="gagne_list">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>80</y>
     <width>381</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
